diff --git a/Dockerfile b/Dockerfile
index 9678fcc..f18e4b7 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,14 +1,14 @@
-FROM alpine:3.12
+FROM amazonlinux:2
 
-RUN apk add bash coreutils findutils iptables curl
+RUN yum install -y bash util-linux coreutils findutils iptables curl
 
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.19.0/bin/linux/amd64/kubectl \
+RUN curl -LO https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubectl \
        && chmod +x ./kubectl \
        && mv ./kubectl /usr/local/bin/kubectl
-ENV KUBECONFIG=/var/lib/k0s/pki/admin.conf
+ENV KUBECONFIG=/var/lib/ekz/pki/admin.conf
 
 ADD docker-entrypoint.sh /entrypoint.sh
-ADD ./k0s /usr/local/bin/k0s 
+ADD ./k0s /usr/local/bin/k0s
 ENTRYPOINT [ "/bin/sh", "/entrypoint.sh" ]
 
 CMD ["k0s", "server", "--enable-worker"]
\ No newline at end of file
